src/main/java/com/onarandombox/MultiverseCore/utils/PermissionsTool.java

```diff
@@ -0,0 +1,329 @@
+package com.onarandombox.MultiverseCore.utils;
+
+import com.dumptruckman.minecraft.util.Logging;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import com.onarandombox.MultiverseCore.api.Destination;
+import com.onarandombox.MultiverseCore.api.MVWorld;
+import com.onarandombox.MultiverseCore.destination.ParsedDestination;
+import org.bukkit.command.CommandSender;
+import org.bukkit.permissions.Permission;
+import org.bukkit.permissions.PermissionDefault;
+import org.bukkit.plugin.PluginManager;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * A utility class for registering and checking permissions.
+ */
+public class PermissionsTool {
```

Comment:
Is this changing how permissions get registered and such? Can you summarize how this is different?

Reply:
not really, its just refactoring the registering of permission to bukkit api from being all scattered between different classes into 1 class.